get_swpc.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+import os
+import sys
+import urllib
+import argparse
+import time
+
+parser = argparse.ArgumentParser(description='Get SWPC web data')
+parser.add_argument('--out', default='.', help="output directory for fetched files")
+args = parser.parse_args()
+
+swpc_urls = {'sgarf.txt': 'https://services.swpc.noaa.gov/text/sgarf.txt',
+             'GOES_xray.gif': 'https://services.swpc.noaa.gov/images/goes-xray-flux.gif',
+             'ACE_5min.gif': 'http://services.swpc.noaa.gov/images/ace-epam-7-day.gif'}
+
+for locname, url in swpc_urls.items():
+    for _ in range(3):
+        try:
+            outfile = os.path.join(args.out, locname)
+            urllib.urlretrieve(url, outfile)
```

Comment:
I don't remember which is the best Python 2 practice for a file fetch

Reply:
OK.  I couldn't remember if they had complimentary functions (as in use requests to actually check status).  requests doesn't seem to implement a get-content-and-write-out function (unless I'm missing something in docs), and writing response.content seemed like it had to be a hack or mistake.